src/vs/workbench/contrib/testing/browser/testingExplorerView.ts

```diff
@@ -417,6 +427,26 @@ export class TestingExplorerView extends ViewPane {
 		}
 	}
 
+	private renderActivityCount() {
+		const countBadgeType = this.configurationService.getValue<TestingCountBadge>('testing.countBadge');
+		let type: 'failed' | 'passed' | 'skipped' | 'totalWillBeRun' | 'off';
```

Comment:
This approach won't localize.

Reply:
Thank you for taking the time to review, can you elaborate more on this?

Reply:
oh ok, I think got it, you mean that `testingCountBadge` won't be localized correctly because the param is directly passed in English. I will change it.

Reply:
When running VS Code in, say, French, the tip will still contain one of these English words.

Reply:
Yes, I missed that, I've just added a localized method that would fix this, would appreciate if you can take another look.

Reply:
LGTM